internal/pb/export/export.proto

```diff
@@ -67,22 +75,41 @@ message TemporaryExposureKey {
   optional bytes key_data = 1;
 
   // Varying risks associated with exposure depending on type of verification
-  optional int32 transmission_risk_level = 2;
+  // Ignored by the v1.5 API when report_type is set.
+  optional int32 transmission_risk_level = 2 [deprecated = true];
 
   // The interval number since epoch for which a key starts
   optional int32 rolling_start_interval_number = 3;
 
   // Increments of 10 minutes describing how long a key is valid
-  optional int32 rolling_period = 4
-      [default = 144];  // defaults to 24 hours
+  optional int32 rolling_period = 4 [default = 144];  // defaults to 24 hours
+
+  // Data type representing why this key was published.
+  enum ReportType {
+    UNKNOWN = 0;  // Never returned by the API.
+    CONFIRMED_TEST = 1;
+    CONFIRMED_CLINICAL_DIAGNOSIS = 2;
+    SELF_REPORT = 3;
+    RECURSIVE = 4;
+    REVOKED = 5;  // Used to revoke a key, never returned by API.
```

Comment:
Is this true - if a key is revoked, will we have a status of REVOKE in the revised_keys list?

Reply:
If we want the client to learn of the revocation, yep.